src/components/webView.re

```diff
@@ -26,6 +26,19 @@ external source :
   source =
   "";
 
+[@bs.obj]
+external sourceHtml :
+  (
+    ~html: string=?,
+    ~baseUrl: string=?,
+    unit
+  ) =>
+  source =
+  "";
+
+[@deprecated "Please use WebView.sourceUri instead"]
```

Comment:
This is only needed in the `rei`. Also did u test it with the RNTester to verify? I never trust reason deprecated annotations :P

Reply:
Ah ok, fixed in dec94b7 (amended to the commit)
And yes the deprecation worked, got warning message
`deprecated: BsReactNative.WebView.source, Please use WebView.sourceUri instead`